src/cmd/embeddedrules/rules.php

```diff
@@ -11,30 +11,50 @@
  * @after   $x ?: $y
  */
 function ternarySimplify() {
-  /**
-   * @maybe could replace the ternary with just $cond
-   * @type bool $cond
-   */
-  $cond ? true : false;
-
-  /**
-   * @maybe could rewrite as `(bool)$cond`
-   * @type  !bool $cond
-   */
-  $cond ? true : false;
-
-  /**
-   * @maybe could rewrite as `$x ?: $y`
-   * @fix $x ?: $y
-   * @pure $x
-   */
-  $x ? $x : $y;
-
-  /**
-   * @maybe could rewrite as `$x ?? $y`
-   * @fix $x ?? $y
-   */
-  isset($x) ? $x : $y;
+    /**
```

Comment:
It seems that auto-formatting happened here. Anyway, I think sooner or later we need to do formatting for the rules 😄

Reply:
Although this would cause merge conflicts, so reverted the formatting back.